src/tokens/eth/0x020C710646e23AB868dbE5B88004892797fE4eFb.json

```diff
@@ -0,0 +1,31 @@
+{
+  "symbol": “GOLF”,
```

Comment:
Change to correct quotes.

Reply:
Not sure what I'm supposed to here. Followed other's that had been verified. Can you help?